src/Wrapper.js

```diff
@@ -116,21 +100,25 @@ export default (Component) => {
       this.context.formsy.detachFromForm(this);
     }
 
-    getErrorMessage() {
+    getErrorMessage = () => {
       const messages = this.getErrorMessages();
       return messages.length ? messages[0] : null;
     }
 
-    getErrorMessages() {
-      return !this.isValid() || this.showRequired() ?
-        (this.state.externalError || this.state.validationError || []) : [];
-    }
+    getErrorMessages = () => {
+      if (!this.isValid() || this.showRequired()) {
+        return this.state.externalError || this.state.validationError || [];
+      }
+
+      return [];
 
-    getValue() {
-      return this.state.value;
+      // !this.isValid() || this.showRequired() ?
+      // (this.state.externalError || this.state.validationError || []) : [];
```

Comment:
?

Reply:
eslint was complaining about ambiguous usage of ternary operators.

Reply:
Ah, I see. Yeah the older code was definitely confusing. This is MUCH cleaner. Thanks! I would delete the commented-out older version and the blank line before the closing bracket. No need to keep it around.